andela/firestar-backend
# #167730733 configure dotenv

#### What does this PR do?
  This PR configures the project to set and use environmental variables using dotenv package

#### Description of Task to be completed?
  - Install dotenv package from npm.
  - Create .env, .env_example files
  - Initailize dotenv in the project entry point
  - Document the use of dotenv in the ReadMe file

#### How should this be manually tested?
  - git clone this using the command `git clone git@github.com:andela/firestar-backend.git` in the terminal
  - cd into the project
  - checkout to the branch using the command `git checkout ch-dotenv-setup-167730733`
  - Run the command `npm install` in the terminal
  - Create a `.env` file and input the variable name (uppercase) and value.
       Example PORT=9003
  - In the .env_example file input the name of the variable without a value
      Example PORT= 
  - Open up a terminal and run the command `npm run start:dev`.
  - Check the console and the PORT number specified as the environment variable will be displayed as seen in the image below.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167730733](https://www.pivotaltracker.com/story/show/167730733)

#### Screenshots (if appropriate)
![Screenshot (3)](https://user-images.githubusercontent.com/46751363/63168671-ae7c8180-c02c-11e9-9f7f-38ccb9f172b5.png)
